app/controllers/projects_controller.rb

```diff
@@ -108,7 +108,7 @@ def create
 
   # PATCH/PUT /projects/1
   # PATCH/PUT /projects/1.json
-  # rubocop:disable Metrics/AbcSize, Metrics/MethodLength
+  # rubocop:disable Metrics/AbcSize, Metrics/MethodLength, Rails/OutputSafety
```

Comment:
We really need to minimize the region where Rails/OutputSafety is disabled.  I believe you can disable & re-enable it *just* around the assignment to flash.  That will also make it much more obvious *why* you're doing that.